neural_speed/models/chatglm/chatglm2.h

```diff
@@ -31,6 +31,12 @@ static const model_scratch chatglm_mem_req(int n_layers, float scratch_size_rati
           static_cast<unsigned long long>(scratch_size_ratio * 2048) * MB,
           static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
       };
+    case 40:
+      return {
+          static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
```

Comment:
could this memory accept 4k input ?  or 2 k input ?

Reply:
better make it to 8K, since this model origin support 8k input

Reply:
![image](https://github.com/intel/neural-speed/assets/105184542/d29e91f1-4302-44be-971b-e6e63bad9678)
here 4k'result, 8k' result is eos token.